spk/jackett/src/service-setup.sh

```diff
@@ -0,0 +1,65 @@
+
+# Package specific behaviors
+# Sourced script by generic installer and start-stop-status scripts
+
+service_prestart ()
+{
+    # Replace generic service startup
+
+    MONO_PATH="${SYNOPKG_PKGDEST}/../mono/bin"
+    PATH="${SYNOPKG_PKGDEST}/bin:${MONO_PATH}:${PATH}"
+    MONO="${MONO_PATH}/mono"
+    JACKETT="${SYNOPKG_PKGDEST}/share/${SYNOPKG_PKGNAME}/JackettConsole.exe"
+    HOME_DIR="${SYNOPKG_PKGDEST}/var"
+
+    echo "Starting Jackett as user ${EFF_USER}" >> ${LOG_FILE}
+    COMMAND="env HOME=${HOME_DIR} PATH=${PATH} LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${MONO} --debug ${JACKETT} --PIDFile ${PID_FILE}"
+
+    if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 6 ]; then
+        su ${EFF_USER} -s /bin/sh -c "cd ${SYNOPKG_PKGDEST}; ${COMMAND}" >> ${LOG_FILE} 2>&1 &
```

Comment:
Is it really needed to change process working directory to SYNOPKG_PKGDEST ? (I did it on `demoservice` to browse files...)
I think HOME_DIR may be more relevant, and it is already done by `su`